paimon-common/pom.xml

```diff
@@ -57,6 +57,14 @@ under the License.
             <version>${airlift.version}</version>
         </dependency>
 
+        <!-- From paimon-format -->
+        <!-- From org.apache.avro -->
+        <dependency>
```

Comment:
You should bundle this in paimon-format

Reply:
Did you mean move commons-compress to paimon-format ?

Reply:
Yes.